plugins/repository-hdfs/build.gradle

```diff
@@ -7,427 +7,388 @@
  */
 
 import org.apache.tools.ant.filters.ReplaceTokens
-import org.apache.tools.ant.taskdefs.condition.Os
 import org.elasticsearch.gradle.internal.info.BuildParams
 import org.elasticsearch.gradle.internal.test.RestIntegTestTask
-
-import java.nio.file.Files
+import org.elasticsearch.gradle.internal.util.HdfsUtils
 import java.nio.file.Path
-import java.nio.file.Paths
 
 import static org.elasticsearch.gradle.PropertyNormalization.IGNORE_VALUE
 
 apply plugin: 'elasticsearch.test.fixtures'
-apply plugin: 'elasticsearch.rest-resources'
-apply plugin: 'elasticsearch.rest-test'
```

Comment:
Ultimately we want to get rid of `rest-test` plugin usage and rely on yamlRestTest and javaRestTest and friends